config-default.yml

```diff
@@ -134,6 +134,7 @@ guild:
 
     staff_channels: [*ADMINS, *ADMIN_SPAM, *MOD_SPAM, *MODS, *HELPERS, *ORGANISATION, *DEFCON]
     ignored: [*ADMINS, *MESSAGE_LOG, *MODLOG]
+    attachment_repost: 649243850006855680
```

Comment:
This should be under the channels with rest of the channel IDs.

Reply:
Agree, and it should be named `attachment_log` to match the in-server channel name.

Reply:
Yep, it was here because it used an anchor node previously, gonna move it

Reply:
Oh, and it's best to add the channel to the `ignored` list just above here